controllers/clusterversion_controller.go

```diff
@@ -54,7 +56,9 @@ const (
 	operatorConfigMapName = "ocs-client-operator-config"
 	// ClusterVersionName is the name of the ClusterVersion object in the
 	// openshift cluster.
-	clusterVersionName = "version"
+	clusterVersionName                          = "version"
+	csiAddonsSubscriptionLabelKeyPrefix         = "operators.coreos.com/odf-csi-addons-operator"
+	ocsClientOperatorSubscriptionLabelKeyPrefix = "operators.coreos.com/ocs-client-operator"
```

Comment:
who is adding the labels? can we go with names as we know it already.

Reply:
The labels are getting added by olm as per my understanding
and regarding known name, the name for csi-addons is `odf-csi-addons-operator-stable-4.15-redhat-operators-openshift-marketplace` where the channel part gets included so its simple to use the label

Reply:
making the change to package name instead of labels as Nitin suggested